src/grafanarmadillo/cmd.py

```diff
@@ -0,0 +1,173 @@
+"""Ready-to-run commands for common Grafana templating scenarios."""
+import json
+from pathlib import Path
+from typing import Dict, Literal, NewType, Union
+
+import click
+from grafana_client import GrafanaApi
+
+from grafanarmadillo.alerter import Alerter
+from grafanarmadillo.dashboarder import Dashboarder
+from grafanarmadillo.find import Finder
+from grafanarmadillo.templator import Templator, findreplace
+
+
+EnvMapping = NewType("EnvMapping", Dict[str, Dict[str, str]])
+Direction = NewType("Direction", Union[Literal["import"], Literal["export"]])
+
+load_file_help = """Should be encoded as json. You can pass this in as a string; or as file using 'file://path/to/file'"""
+mapping_help = \
+	"Mapping of values to findreplace in the template. " + \
+	"The type should be a mapping of environment names (like 'template' or 'production') " + \
+	"to dicts of replacements (for example, `{\"region\": \"SouthWest\"}`). " + \
+	load_file_help
+env_grafana_help = "Name of the environment in the mapping file for Grafana, found in the `--mapping` argument"
+env_template_help = "Name of the environment in the mapping file for the template, found in the `--mapping` argument"
+
+
+def load_data(data_str: str):
+	"""Attempt to load data."""
+	_file_uri_prefix = "file://"
+	if data_str.startswith(_file_uri_prefix):
+		filename = Path(data_str.split(_file_uri_prefix)[1])
```

Comment:
I really don't like the split approach for this stuff, shame it's not `3.9+`, cause then you could just throw in a `str.removeprefix`.. almost worth using something like `data_str[len(_file_uri_prefix):]`, but that's also terrible

Reply:
yeah I should probably just urlparse it